handler/src/main/java/io/netty/handler/ssl/SslUtils.java

```diff
@@ -293,13 +298,17 @@ static int getEncryptedPacketLength(ByteBuf buffer, int offset) {
         if (tls) {
             // SSLv3 or TLS or GMSSLv1.0 or GMSSLv1.1 - Check ProtocolVersion
             int majorVersion = buffer.getUnsignedByte(offset + 1);
+            int version = buffer.getShort(offset + 1);
             if (majorVersion == 3 || buffer.getShort(offset + 1) == GMSSL_PROTOCOL_VERSION) {
```

Comment:
```suggestion
            if (majorVersion == 3 || version == GMSSL_PROTOCOL_VERSION) {
```

Reply:
Didn't notice GMSSL also read it as a short. Will do.